vault/external_tests/seal_migration/seal_migration_pre14_test.go

```diff
@@ -0,0 +1,134 @@
+// +build !enterprise
+
+package seal_migration
```

Comment:
May I suggest using `sealmigration` as the package name instead? [According to this blog post](https://blog.golang.org/package-names), package names shouldn't have `_` (or be `camelCased`).

Reply:
good idea, done